test/hack/cleanup/main.go

```diff
@@ -54,13 +56,18 @@ type CleanableResourceType interface {
 	Type() string
 	Get(context.Context, time.Time) ([]string, error)
 	Cleanup(context.Context, []string) ([]string, error)
+	DeleteByClusterName(context.Context, string) error
 }
 
 type MetricsClient interface {
 	FireMetric(context.Context, string, float64, string) error
 }
 
 func main() {
+	var clusterName string
+	if len(os.Args) != 0 {
```

Comment:
Should we check if the length of the args is exactly what we expect (0 or 1). Maybe worth using a flag here to be more explicit as opposed to OS args?